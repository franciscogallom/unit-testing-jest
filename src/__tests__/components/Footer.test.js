import React from 'react';
import { mount } from 'enzyme';
import { create } from 'react-test-renderer';
import Footer from '../../components/Footer';

describe('<Footer />', () => {
  const footer = mount(<Footer />);
  test('component rendering', () => {
    expect(footer.length).toEqual(1);
  });
  test('title rendering', () => {
    expect(footer.find('.Footer-title').text()).toEqual('Platzi Store');
  });
});

describe('Footer Snapshot', () => {
  test('check the UI of the Foother component', () => {
    const footer = create(<Footer />);
    expect(footer.toJSON()).toMatchSnapshot();
  });
});
