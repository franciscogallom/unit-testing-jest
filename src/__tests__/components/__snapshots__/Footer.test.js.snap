// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Snapshot check the UI of the Foother component 1`] = `
<div
  className="Footer"
>
  <p
    className="Footer-title"
  >
    Platzi Store
  </p>
  <p
    className="Footer-copy"
  >
    Todos los Izquierdos Reservados
  </p>
</div>
`;
